analyzer/consensus/consensus.go

```diff
@@ -33,20 +33,10 @@ const (
 	registryUpdateFrequency = 100 // once per n block
 )
 
-var (
-	// ErrOutOfRange is returned if the current block does not fall within tge
-	// analyzer's analysis range.
-	ErrOutOfRange = errors.New("range not found. no data source available")
-
-	// ErrLatestBlockNotFound is returned if the analyzer has not indexed any
-	// blocks yet. This indicates to begin from the start of its range.
-	ErrLatestBlockNotFound = errors.New("latest block not found")
-)
-
 // Main is the main Analyzer for the consensus layer.
 type Main struct {
```

Comment:
Ah! This was actually what it was called initially. But our linter complained about redundant naming, since if you import this package anywhere, you must reference `consensus.ConsensusAnalyzer`. So it felt cleaner to reference `consensus.Main` after import.